met-web/src/components/engagement/form/EngagementWidgets/AddContactDrawer.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useContext, useState } from 'react';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import Divider from '@mui/material/Divider';
+import { Grid } from '@mui/material';
+import { MetHeader3, MetLabel, PrimaryButton, SecondaryButton } from 'components/common';
+import ImageUpload from 'components/imageUpload';
+import { useForm, FormProvider, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import ControlledTextField from 'components/common/ControlledInputComponents/ControlledFormInput';
+import { postContact } from 'services/contactService';
+import { useAppDispatch } from 'hooks';
+import { openNotification } from 'services/notificationService/notificationSlice';
+import { WidgetDrawerContext } from './WidgetDrawerContext';
+
+const schema = yup
+    .object({
+        name: yup.string().required(),
```

Comment:
how do we handle picture of the contact? i mean the display picture

Reply:
Not handled yet, it's a placeholder for now. When it will be implemented later, it will be handled like the banner in the engagement form. It will be separate for the different parts of the form and handled separately